### PR TITLE
CI on PRs

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
## Pull Request Info
Adding test runs to our PRs so we can ensure that:
1) Tests are frequently run (as often as we PR, at least)
2) New changes do not break existing tests

See the Actions page: https://github.com/mongodb/docs-realm/actions of this repo to see our tests. Currently iOS, .NET, and Android pass reliably; Node.js passes most tests, but often fails because of async callbacks.

### CODE REVIEW

File: examples/dotnet/Examples/EmbeddedExamples.cs [hunk 1]
```diff
@@ -156,8 +157,8 @@ public async Task QueryEmbeddedObject()
                 }
                 //:code-block-end:
 
-                /* Test that the query worked and that the Contacts returned 
-                 * actually are from 'Los Angeles'. */
+                // Test that the query worked and that the Contacts returned 
+                // actually are from 'Los Angeles'.
```
Comment (MongoCaleb) [inline examples/dotnet/Examples/EmbeddedExamples.cs:161]: Why this change? Doesn't matter, just curious.

File: examples/dotnet/Examples/EmbeddedExamples.cs [hunk 2]
```diff
@@ -187,16 +188,16 @@ public class Address : EmbeddedObject
             [MapTo("_partition")]
             public string Partition { get; set; }
 
-            [MapTo("street")]
+            //[MapTo("street")]
```
Comment (MongoCaleb) [inline examples/dotnet/Examples/EmbeddedExamples.cs:191]: These need to be in here and not commented out. They are part of a bluehawk code example and are important.

Comment (MongoCaleb) [reply examples/dotnet/Examples/EmbeddedExamples.cs:191]: What's the error when you don't comment them out?

**Review by MongoCaleb — COMMENTED** [review]
One blocking change; otherwise lgtm

File: examples/dotnet/Examples/EmbeddedExamples.cs (hunk 2 above)
Comment (nathan-contino-mongo) [reply examples/dotnet/Examples/EmbeddedExamples.cs:191]: It seems the mapped fields don't actually match the field names in the schema -- if we just fix the backend schema (to use lowercase letters) we should be able to throw these changes away.

File: examples/dotnet/Examples/EmbeddedExamples.cs (hunk 1 above)
Comment (nathan-contino-mongo) [reply examples/dotnet/Examples/EmbeddedExamples.cs:161]: Originally I had to comment out this whole test, and you can't nest block comments in comments (the outer block comment ends when the inner block comment ends, leaving you with a hanging block comment end somewhere later on. Definitely can change this back too!

**Review by MongoCaleb — APPROVED** [review]
LGTM

**Review by cbush — APPROVED** [review]
